index.bs

```diff
@@ -1887,49 +1768,26 @@ const a = builder.input('a', descA);
 const descB = {type: 'float32', dimensions: [4, -1]};
 const b = builder.input('b', descB);
 const c = builder.matmul(a, b);
-const graph = await builder.build({c});
+const graph = builder.build({'c': c});
 
-async function compute(shapeA, shapeB) {
+function compute(shapeA, shapeB, shapeC) {
```

Comment:
Should we also use a different name for this custom `compute` function that wraps the `MLGraph.compute`? It is doing both buffer allocation and compute.

Reply:
Good idea. I renamed it to `allocateAndCompute` in the new commit. Please take a look.